tooltester/pom.xml

```diff
@@ -408,7 +408,6 @@
         <dependency>
             <groupId>org.yaml</groupId>
             <artifactId>snakeyaml</artifactId>
-            <version>1.33</version>
```

Comment:
This should resolve the issue in this ticket: https://github.com/dockstore/dockstore-support/pull/460